torch/quantization/fx/quantize.py

```diff
@@ -1313,7 +1313,20 @@ def record_match(pattern, node, matched):
                                 # handler logic to take this into account.
                                 value = CopyNodeQuantizeHandler  # type: ignore
 
-                            this_node_qconfig = self.qconfig_map[node.name]
+                            # to properly check for dtype support, we need to
```

Comment:
is this the case when base_node is configured to be quantized and relu is not?

Reply:
this is because dtypes supported by relu can be different than dtypes supported by add/mul/sub/etc, and since we are in `BinaryOpQuantizeHandler`, the dtypes to use need to be the ones of the base node.